This patch is needed to fix usage of faac with track numbers higher than 7 with
a 0 prefix. The original scanf parameter %i will consider numbers starting by 0
as octal, thus breaking the numbering for 08 and 09 tracks.

See Gentoo bug #80690.

Index: faac/frontend/main.c
===================================================================
--- faac.orig/frontend/main.c
+++ faac/frontend/main.c
@@ -607,10 +607,10 @@ int main(int argc, char *argv[])
 	    album = optarg;
 	    break;
 	case TRACK_FLAG:
-	    sscanf(optarg, "%i/%i", &trackno, &ntracks);
+	    sscanf(optarg, "%d/%d", &trackno, &ntracks);
 	    break;
 	case DISC_FLAG:
-	    sscanf(optarg, "%i/%i", &discno, &ndiscs);
+	    sscanf(optarg, "%d/%d", &discno, &ndiscs);
 	    break;
 	case COMPILATION_FLAG:
 	    compilation = 0x1;
